lib/ansible/modules/network/nxos/nxos_hsrp.py

```diff
@@ -165,14 +143,10 @@ def execute_show_command(command, module, command_type='cli_show'):
 
 def apply_key_map(key_map, table):
     new_dict = {}
-    for key, value in table.items():
+    for key in table:
         new_key = key_map.get(key)
         if new_key:
-            value = table.get(key)
-            if value:
-                new_dict[new_key] = str(value)
-            else:
-                new_dict[new_key] = value
+            new_dict[new_key] = str(table.get(key))
```

Comment:
Reverted the change https://github.com/ansible/ansible/pull/24770/commits/10fac90ad602745853b70cfdc0692103b4db25ec as it might return None https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/nxos/nxos_hsrp.py#L249-L258